src/interp.cc

```diff
@@ -1358,6 +1360,24 @@ ValueTypeRep<T> SimdSplat(V lane_data) {
   return ToRep(Bitcast<T>(simd_data));
 }
 
+// Simd instructions with Lane operand.
+// a0: input v128 value. a1: lane index. 
+// typename T: lane data type.
+template <typename R, typename V0, typename V1, typename T>
+ValueTypeRep<R> SimdExtractLane(V0 a0, V1 a1) {
```

Comment:
Better name for `a1`? Perhaps `lane` or `index`.

Reply:
Agree.

Reply:
Done.